lib/team.rb

```diff
@@ -6,7 +6,7 @@ class Team
               :name,
               :abbreviation,
               :link
-  @@all_teams = []
+  @@all = []
 
```

Comment:
I think for now this should be switched back to @@all_teams unless we decide to make a Readable module